src/components/ui/LoadingSkeletons.tsx

```
@@ -716,6 +716,256 @@ export const RacesPageSkeleton = () => (
   </div>
 )
 
+// Weekly Planner Page Skeleton
+export const WeeklyPlannerSkeleton = () => (
+  <div
+    className="max-w-[1600px] mx-auto px-4 lg:px-8 py-4 lg:py-8"
+    role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)